### PR TITLE
DR-2344 Represent null and empty strings differently

### PR DESCRIPTION
**Pull request by tlangs**
Representing empty strings and null values differently in data tables makes it easier for users to track down what's in their datasets. 

The following screenshot is the result of ingesting:
```json
{"sample_name":"emptyDataType","data_type":""}
{"sample_name":"nullDataType","data_type":null}
{"sample_name":"missingDataType"}
```

image

### CODE REVIEW

**Review by snf2ye — APPROVED** [review]

File: src/components/dataset/query/DatasetQueryView.jsx [hunk 1]
```diff
@@ -213,7 +213,7 @@ export class DatasetQueryView extends React.PureComponent {
               <QueryViewDropdown options={names} onSelectedItem={this.handleChange} />
             </Grid>
             <Grid item xs={3}>
-              <Link to="details">
+              <Link to="overview">
```
Comment (nmalfroy) [inline src/components/dataset/query/DatasetQueryView.jsx]: huh...I thought we fixed this already

File: src/components/table/JadeTable.jsx [hunk 2]
```diff
@@ -126,25 +131,34 @@ export class JadeTable extends React.PureComponent {
   };
 
   handleArrayValues = (value, column) => {
+    const returnValue = [];
+    if (column.mode === COLUMN_MODES.REPEATED) {
+      returnValue.push(<span key="start">[</span>);
+    }
+    returnValue.push(
+      ...value.flatMap((v, i) => [
+        v,
```
Comment (nmalfroy) [inline src/components/table/JadeTable.jsx]: what if the value in the array is null?  I think you want
```suggestion
        _.isNull(v) ? this.handleNullValue(v) : v,
```

**Review by nmalfroy — CHANGES_REQUESTED** [review]
nice!  I do wonder about nulls in arrays?

File: src/components/dataset/query/DatasetQueryView.jsx (hunk 1 above)
Comment (tlangs) [reply src/components/dataset/query/DatasetQueryView.jsx]: We did! Just updated with contents from develop

**Review by nmalfroy — APPROVED** [review]
thanks for making the changes!

**Pull request merged**